test/e2e_node/container_manager_test.go

```diff
@@ -192,6 +195,9 @@ var _ = SIGDescribe("Container Manager Misc [Serial]", func() {
 						},
 					},
 				})
+				if err := e2epod.WaitForPodCondition(f.ClientSet, f.Namespace.Name, podName, "Ready", 120*time.Second, testutils.PodRunningReady); err != nil {
+					framework.Logf("Pod %v took too long to enter running/ready: %v", podName, err)
```

Comment:
I'd suggest to fail the test in this case. Otherwise we are checking the oom score adj for the wrong process potentially

Reply:
Done